crates/circuit/src/dag_circuit.rs

```diff
@@ -81,6 +81,15 @@ pub enum NodeType {
     Operation(PackedInstruction),
 }
 
+impl NodeType {
+    pub fn unwrap_operation(&self) -> &PackedInstruction {
+        match self {
+            NodeType::Operation(instr) => instr,
+            _ => panic!("Node is not an operation!"),
```

Comment:
Can we add a docstring to this to document why this method exists and why it panics? I know it's a little redundant, but it would be convenient so a developer can know just from the docstring alone.

Reply:
Done in 6245eeb344e491211fbf29d51b0c8defe60df0f6